test/SlowTests/Issues/RavenDB_16358.cs

```diff
@@ -57,10 +57,10 @@ public async Task AggressiveCacheWithTimeoutTestAsync()
             Assert.Equal(requests + 1, requestExecutor.NumberOfServerRequests);
         }
 
-        private async Task LoadDataAsync(IDocumentStore store)
+        private static async Task LoadDataAsync(IDocumentStore store)
         {
             using (var session = store.OpenAsyncSession())
-            using (session.Advanced.DocumentStore.AggressivelyCacheFor(TimeSpan.FromMilliseconds(300)))
+            using (await session.Advanced.DocumentStore.AggressivelyCacheForAsync(TimeSpan.FromMilliseconds(300)))
```

Comment:
By the way this is a classical pitfall of returning `Task<IDisposable>` and I bet this will lead to many support hazzles.